pkg/resolver/resolver.go

```diff
@@ -0,0 +1,61 @@
+package resolver
```

Comment:
Putting this package inside the `did` package would make more sense imo. "Resolver" lacks context